### ISSUE TITLE
Flag projects that have no a11y issues

### ISSUE THREAD

**Issue by nkkl** [18F/a11y-metrics#4]
So that we can see projects that appear to be missing a11y completely.

Exception: if there are no issues at all.

**Comment by toolness** [comment]
Good idea--I assume by "issues" you mean GitHub issues? When I first read the title of this issue, I thought you meant "issues" in the sense of "problems", which confused me.  I've renamed this issue to reflect that but feel free to change it if I misinterpreted!

**Comment by nkkl** [comment]
Your interpretation is in line with what I intended. :)
